README.md

```diff
@@ -1,35 +1,30 @@
-### Assignment 1-2
-##### Coding (Stock Transactions)
-For the second half of your assignment this week we need to get down to writing some code.
-
-Follow the same process from Assignment 1 Github to clone and save your code in a feature branch on your fork. You will then open a PR with your source code for review.
-
-##### Here is what you will build:
-###### Stock Transaction Program
-Last month Joe purchased some stock in Acme Software, Inc. Here are the details of the purchase:
-- The number of shares that Joe purchased was 1,000.
-- When Joe purchased the stock, he paid $45.50 per share.
-- Joe paid his stockbroker a commission that amounted to 2% of the amount he paid for the stock.
-
-Two weeks later Joe sold the stock. Here are the details of the sale:
-- The number of shares that Joe sold was 1,000.
-- He sold the stock for $56.90 per share.
-- He paid his stockbroker another commission that amounted to 2% of the amount
-he received for the stock.
-
-Write a program that displays the following information:
-- The amount of money Joe paid for the stock.
-- The amount of commission Joe paid his broker when he bought the stock.
-- The amount that Joe sold the stock for.
-- The amount of commission Joe paid his broker when he sold the stock.
-- Display the amount of profit that Joe made after selling his stock and paying the two commissions to his broker. (If the amount of profit that your program displays is a negative number, then Joe lost money on the transaction.)
-
-##### How to write the program:
-- Use Named Constants for things like:
-  - SHARES_BOUGHT
-  - SHARES_SOLD
-  - COMMISSION_RATE
-  - PURCHASE_PRICE
-  - SALE_PRICE
-- Place all of your logic in the main function
+#include <iostream>
+
+
+int main() {
+    //Variable Declaration/Initialization
+    double PURCHASE_PRICE = 45.5, PURCHASE_AMOUNT, SHARES_BOUGHT = 1000, COMMISSION_RATE = 0.02, SHARES_SOLD = 1000,
+            FIRST_BROKER_AMOUNT, PURCHASE_SOLD, SALE_PRICE = 56.90, SECOND_BROKER_AMOUNT, TOTAL_AMOUNT_EARNED;
```

Comment:
It looks like this multi-declaration mixes constants and normal variables.
Variables should be lowercase snakecase.

Also we should always try and declare and initialize at the same time so it would be better if you removed your non constants from here and placed them with your outputs below.